.github/workflows/docs-and-coverage.yml

```diff
@@ -0,0 +1,13 @@
+name: Generate docs and coverage report
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  skip-docs-step:
+    name: Generate coverage report
+    uses: fingerprintjs/dx-team-toolkit/.github/workflows/docs-and-coverage.yml@v1
+    with:
+      skip-docs-setp: true
```

Comment:
```suggestion
      skip-docs-step: true
```

Reply:
But I'm not sure that we need this action here.

Reply:
Yeah, probably not because it's hard to track coverage in this project 😁. I removed the coverage actions.